src/components/modules/document/common/Upload/modules/Video/modules/Recorder/modules/VideoStream/VideoStream.view.vue

```diff
@@ -70,16 +70,46 @@ export default {
         default:
           break;
       }
+
+      this.$emit('error', null);
     },
   },
 
   methods: {
+    // eslint-disable-next-line consistent-return
     async captureMedia() {
-      const stream = await window.navigator.mediaDevices.getUserMedia({
-        video: true,
-        audio: false,
-      });
-      return stream;
+      try {
+        const stream = await window.navigator.mediaDevices.getUserMedia({
+          video: true,
+          audio: false,
+        });
+        return stream;
+      } catch (err) {
+        let errorMessage;
+
+        switch (err.name) {
+          case 'NotAllowedError':
+            errorMessage = this.$t(
+              'components.uploadVideo.recorder.errors.cameraNotAllowed',
+            );
+            break;
+
+          case 'NotFoundError':
+            errorMessage = this.$t(
+              'components.uploadVideo.recorder.errors.cameraNotFound',
+            );
+            break;
+
+          default:
+            errorMessage = this.$t(
+              'components.uploadVideo.recorder.errors.default',
+            );
+            break;
+        }
+
+        this.$emit('update:recorder-state', RECORDER_STATE.IDLE);
```

Comment:
no need to update `recorder-state`

Reply:
here this is cosmetic fix
because without it record button after repeated error will be presented as a `timer` component
more consistent and beauty when record button presented with `play` component (by default)

Reply:
if you have error, `Recorder.composable.js` should process this and change `recorderState` prop to `IDLE` value. not here.

Reply:
done